src/ReflectionFileNamespace.php

```diff
@@ -357,6 +362,20 @@ private function findConstants()
             }
         }
 
+        if ($withDefinedConstants) {
+            foreach ($this->namespaceNode->stmts as $namespaceLevelNode) {
+                if ($namespaceLevelNode instanceof FuncCall && (string)$namespaceLevelNode->name === 'define') {
```

Comment:
What PHP code should look like to have `$namespaceLevelNode->name` property of `Expr` class instead of `Name` class?